modules/installation-complete-user-infra.adoc

```diff
@@ -90,9 +90,7 @@ service-ca                                 4.7.0   True        False         Fal
 storage                                    4.7.0   True        False         False      4h30m
 ----
 +
-When all of the cluster Operators are `AVAILABLE`, you can complete the installation.
-
-. Monitor for cluster completion:
+Alternatively, the following command notifies you once all the of the clusters are available as well as retrieves and displays credentials:
```

Comment:
Alternatively, the following command notified you when all of the clusters are available. It also retrieves and displays credentials: